vividus/src/main/java/org/vividus/bdd/CompositeStepReporter.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2019-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.bdd;
+
+import javax.inject.Inject;
+
+import org.vividus.bdd.context.BddRunContext;
+
+public class CompositeStepReporter extends ChainedStoryReporter
```

Comment:
the logic of this class should be fully merged into `org.vividus.bdd.RunContextStoryReporter`